Gemfile.lock

```diff
@@ -50,9 +50,8 @@ GIT
 GEM
   remote: https://gems.contribsys.com/
```

Comment:
I did bundle update using bundler 2.2.27 that we put in this commit https://github.com/pulibrary/bibdata/pull/1766/files#diff-89cade48462044ee1b672dc5f4c3ec250fbd29effcd8932096a23c1283c6731fR641